internal/stack/todo.go

```diff
@@ -0,0 +1,8 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package stack
+
+// TODO compose options
+// TODO stack options
```

Comment:
Is this file intentional?

Reply:
I forgot to remove this one, all TODO items are done.